golang/cosmos/go.mod

```diff
@@ -198,7 +209,7 @@ replace (
 	// use cometbft
 	// Use our fork at least until post-v0.34.14 is released with
 	// https://github.com/tendermint/tendermint/issue/6899 resolved.
-	github.com/tendermint/tendermint => github.com/agoric-labs/cometbft v0.34.30-alpha.agoric.2
+	github.com/tendermint/tendermint => github.com/agoric-labs/cometbft v0.34.35-alpha.agoric.2
```

Comment:
I would feel more confident tagging a `v0.34.35-alpha.agoric.3`

Reply:
Oh, I suppose having the CI green is a good enough reason to tag a new CometBFT sub-version.